docker-compose-generator/docker-fragments/opt-add-bluewallet-lndhub.yml

```diff
@@ -0,0 +1,33 @@
+version: "3.7"
```

Comment:
I dont think will work. The docker fragments are combined into one so if you have any features that do not fit the current schema, it will break

Reply:
ah, it is working for me like that, Ill remove it

Reply:
done